ports/raspberrypi/Makefile

```diff
@@ -171,7 +172,7 @@ else
 endif
 
 # Remove -Wno-stringop-overflow after we can test with CI's GCC 10. Mac's looks weird.
-DISABLE_WARNINGS = -Wno-stringop-overflow -Wno-cast-align
+DISABLE_WARNINGS = -Wno-stringop-overflow -Wno-cast-align -Wno-undef
```

Comment:
We prefer to keep our code free of `-Wundef` diagnostics by leaving the flag as-is for most files. My [comment](https://github.com/adafruit/circuitpython/pull/8091#issuecomment-1599320552) suggested how to apply the `-Wno-undef` compiler flag only to files within the SDK. Did you try this?

Reply:
I went ahead and fixed this as I intended. The suggestion in my earlier comment had an error that was probably not easy to spot, so it wouldn't have worked as-is.